python/pyproject.toml

```diff
@@ -34,6 +35,7 @@ python = ">=3.9,<4.0"
 pyarrow = "^12.0.1"
 typing-extensions = "^4.7.1"
 plotly = {version = "^5.16.1", optional = true}
+pydot = { version = "^1.4.2", optional = true }
```

Comment:
There are other crates we could use. Not sure which is best.

https://pygraphviz.github.io/documentation/stable/ has newer releases. Maybe we should switch?

Either way, it's an optional dependency, so you don't need to have it.

Reply:
@jordanrfrazier any thoughts on dependencies / packages? I think they'll all accomplish the same end goal.

Reply:
Both `graphviz` and `pydot` look like good options to me. `pygraphviz` looks like the heavier-weight option with manipulation of dot files, that I don't believe we need. 

https://github.com/xflr6/graphviz

Not sure if there's significant difference between graphviz and pydot. Both lightweight, both used about the same, but pydot is pure python, (whereas graphviz is a python interface to the graphviz c library?), not sure if that's a factor to lean towards pydot.

Reply:
Switch to `graphviz` -- it has a return type that automatically renders the SVG, but also lets you interact with it to save as a file of different formats, etc. It also seems newer / more maintained.